src/server/pps/server/monitor.go

```diff
@@ -70,14 +70,8 @@ func (m *ppsMaster) startMonitor(pipelineInfo *pps.PipelineInfo) {
 			"monitorPipeline for "+pipeline, func(pachClient *client.APIClient) {
 				// monitorPipeline needs auth privileges to call subscribeCommit and
 				// blockCommit
-				// TODO(msteffen): run the pipeline monitor as the pipeline user, rather
-				// than as the PPS superuser
-				if err := m.a.sudo(pachClient, func(superUserClient *client.APIClient) error {
-					m.monitorPipeline(superUserClient, pipelineInfo)
-					return nil
-				}); err != nil {
-					log.Errorf("error monitoring %q: %v", pipeline, err)
-				}
+				pachClient.SetAuthToken(ptr.AuthToken)
+				m.monitorPipeline(pachClient, pipelineInfo)
```

Comment:
I like that this is shorter than the code that had the TODO on it. It's like, `// TODO: delete this todo`